package.json

```diff
@@ -1,75 +1,56 @@
 {
-  "name": "doks-child-theme",
-  "description": "Doks child theme",
-  "version": "0.5.0",
-  "engines": {
-    "node": ">=16.16.0"
-  },
-  "browserslist": [
-    "defaults"
-  ],
-  "repository": "https://github.com/h-enk/doks",
+  "name": "passkeys.dev",
+  "version": "0.0.0",
+  "description": "Doks theme",
+  "author": "Hyas",
```

Comment:
nit: At this point we should customize this too since the site's established enough.